src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  line-height: 14px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  object-fit: cover;
+  margin-left: 50px;
+
+}
+
+.logo-link {
+  display: block;
+  width: 40px;
+  height: 40px;
+  object-fit: cover;
+}
+
+.nav-list {
+  text-transform: uppercase;
+  margin-right: 40px;
+}
+
+.list {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+.list > li {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 10px;
+}
+
+.menu-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 23px 0 19px;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 1);
+}
+
+.down-line {
+  height: 4px;
+  width: 100%;
+  background-color: rgba(0, 172, 220, 1);
```

Comment:
Move all repeatable colors to CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)